common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -4,7 +4,8 @@
 @import templates.inlineJS.blocking.js._
 
 <!--[if lt IE 9]>
-    <script src="@Static("javascripts/components/html5shiv/html5shiv.js")"></script>
+    <script src="@Static("javascripts/ie8polyfills.js")"></script>
+    <script>@Html(polyfills().body)</script>
```

Comment:
might as well move json to its own conditional comment? seeing as we're in there tidying it all up?

```html
<!--[if lt IE 8]>
    <script src="@Static("javascripts/components/JSON-js/json2.js")"></script>
<![endif]-->
```